src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -443,11 +443,15 @@ When using the `<ViewTransitions />` router, the following steps occur to produc
 7. The router executes any new scripts added to the page.
 8. The `astro:page-load` event fires. This is the end of the navigation process.
 
-## Script behavior during page navigation
+## Script behavior
 
 When navigating between pages with the `<ViewTransitions />` component, scripts are run in sequential order to match browser behavior. 
 
-If you have code that sets up global state, this state will need to take into account that the script might execute more than once. Check for the global state in your `<script>` tag, and conditionally execute your code where possible:
+Bundled module scripts, which are the default scripts in Astro, only ever execute once. If a bundled module script exists on pages that are transitioned to after its initial execution they will be ignored. Inline scripts, ones with `is:inline` attribute, will not reexecute if they exist on both the page being transitioned to and the one the user is currently on. 
+
+Unlike bundled module scripts, inline scripts have the potential to be reexcuted during a user's visit to a site, if they exist on a page that is visited multiple times. Inline scripts might also reexecute if they exist on some pages but not all, if a user goes to a page without the script then back to one with the script it will be reexecuted.
+
+If you have code that sets up global state in an inline script, this state will need to take into account that the script might execute more than once. Check for the global state in your `<script>` tag, and conditionally execute your code where possible. This works because `window` is preserved.
```

Comment:
A few grammar/typo nits. From what I saw the correct spelling for re-execute is using a hyphen.
```suggestion
Bundled module scripts, which are the default scripts in Astro, are only ever executed once. If a bundled module script exists on pages that are transitioned to after its initial execution they will be ignored. Inline scripts, ones with `is:inline` attribute, will not re-execute if they exist on both the page being transitioned to and the one the user is currently on. 

Unlike bundled module scripts, inline scripts have the potential to be re-executed during a user's visit to a site, if they exist on a page that is visited multiple times. Inline scripts might also re-execute if they exist on some pages but not all, if a user goes to a page without the script then back to one with the script it will be re-executed.

If you have code that sets up a global state in an inline script, this state will need to take into account that the script might execute more than once. Check for the global state in your `<script>` tag, and conditionally execute your code where possible. This works because `window` is preserved.
```